### ISSUE TITLE
Can't get it working

### ISSUE THREAD

**Issue by Demoniko** [ionide/ionide-vscode-fsharp#7]
I just installed Ionide-fsharp package and can't get it working. It doesn't work with single files - it seems instead it uses built-in syntax highlighter, same for projects - I made test project with yo fsharp and opened folder with vscode with 0 success. Am I doing something wrong?

**Comment by Demoniko** [comment]
Also forgot - FSI works for me (I have FSI commands in commands menu - start fsi, send line, send selection etc)

**Comment by danslapman** [comment]
I'd faced similiar issue just now, copletion just said "Loading..." and nothing more. The reason caused this issue was blocking FsAutoComplete.Suave.exe by antivirus software. If You are on windows, check this, it may help.

**Comment by Demoniko** [comment]
I do not have antivirus software other than what comes with windows 10, does it also block it?

**Comment by ad1ttya** [comment]
I'm facing a similar issue on Windows 10.. Ionide fsharp version 0.3.1 was working well, after updating to 1.0.2 [1], the editor switched back to built-in syntax highlighter and the FSI commands returned a error/warning "no handler for fsi.Start". Fortunately I had the files of 0.3.1 on my pc, so I replaced them and got things working back again.

[1]: I wasn't able to update directly from 0.3.1 to 1.0.2. I had to uninstall the editor. Remove extensions from 'C:\users\<username>\.vscode\extensions' and then try installing again.
